README.md

```diff
@@ -37,19 +37,21 @@ not the actual file-system path).
 ##### dotfiles
 
 Set how "dotfiles" are treated when encountered. A dotfile is a file
-or directory that begins with a dot ("."). Note this check is done on
-the path itself without checking if the path actually exists on the
-disk. If `root` is specified, only the dotfiles above the root are
-checked (i.e. the root itself can be within a dotfile when when set
-to "deny").
+or directory that begins with a dot (".").
 
+Note: This check is performed on the url pathname.
+It doesn't affect `root` and doesn't actually check if the path existence on disk.
```

Comment:
Can you clarity what "It doesn't affect `root`" means? To me, as even the author, this statement doesn't make sense. If it did "affect root", I would presume that it would have altered that property's value?